AutomatedTesting/Levels/Terrain/TerrainPhysicsCollider_MaterialMapping_Works/tags.txt

```diff
@@ -0,0 +1,12 @@
+0,0,0,0,0,0
```

Comment:
The tags.txt file in level folder is more of a byproduct that is not really necessary. Can you verify whether the test still works if you delete this? If it does, we can commit one less file to the project.

Reply:
Seems to work just fine. Removed!